Source/Common/GLTFUnarchiver.swift

```diff
@@ -1298,20 +1298,32 @@ public class GLTFUnarchiver {
         group.animations = animations
         group.repeatCount = .infinity
         
-        let step = animations.count
-        let dataLength = values.count / step
-        guard dataLength == keyTimes.count else {
-            throw GLTFUnarchiveError.DataInconsistent("loadWeightAnimationsSampler: data count mismatch: \(dataLength) != \(keyTimes.count)")
-        }
-        for i in 0..<animations.count {
-            var valueIndex = i
-            var v = [NSNumber]()
-            v.reserveCapacity(dataLength)
-            for _ in 0..<dataLength {
-                v.append(values[valueIndex])
-                valueIndex += step
+        if keyTimes.count == values.count {
+            for i in 0..<animations.count {
+                var v = [NSNumber]()
+                v.reserveCapacity(values.count)
+                for j in 0..<values.count {
+                    v.append(values[j])
+                }
+                animations[i].values = v
+            }
+        } else {
+            let step = animations.count
+            let dataLength = values.count / step
+            // Comment out to fix animations
+//            guard dataLength == keyTimes.count else {
```

Comment:
nit: indentation

Reply:
done

Reply:
@ox meant that these comments should start in the same column  as the `let` above

```suggestion
            // guard dataLength == keyTimes.count else {
```

Reply:
I commented it with special Xcode command. I think it more better

Reply:
please explain

Reply:
I didn't write "//" manually. I used Xcode hotkey. When you use this hotkey Xcode puts slashes on start of line (hotkey: CMD+/)